### PR TITLE
Mapping one operationplan to one demand if allowsplits = false and bucketized resources

### PR DESCRIPTION
**Pull request by kobsam**

### CODE REVIEW

File: include/frepple/model.h [hunk 1]
```diff
@@ -2585,6 +2590,9 @@ class OperationPlan : public Object,
   /* Quantity. */
   double quantity = 0.0;
 
+  /* Demand name to identify which demand takes this operationplan */
+  string demandReference = "";
```
Comment (jdetaeye) [inline include/frepple/model.h:2594]: Can't you use the existing batch field for this? 
Operationplan is one of the biggest memory consumers in a big modeL  I'll be very hesitant to merge a pull a request that adds a string to every instance.

**Comment by jdetaeye** [comment]
Wouldn't the make-to-order functionality using the batch field not work for this?   See https://frepple.org/docs/current/user-guide/examples/buffer/make-to-order.php 

If all your items are marked of type "make-to-order" and you set the batch field of your demands, then frepple automatically constructs a supply path that is dedicated to each demand.  In such a model, manufacturing orders will automatically have a hard association with a demand.

I haven't checked the code changes in detail, but at first glance this risks to break a lot of existing models and not backwards compatible.

**Comment by kobsam** [comment]
The batch on the demand is just used in the delivery-operation as I see. But i need the operationplan (manufactoring order) that is created for each demand and for that I created this . I work with bucketized resources and allowSplits = false.

**Comment by jdetaeye** [comment]
Sorry, won't merge this.

**Pull request closed without merging**